scripts/generate/generate.sh

```diff
@@ -56,7 +58,12 @@ usage: $(basename "$0") [-b|--branch] [-c|--config-path] [-d|--directive-map-nam
     
     -p  | --path                Path to a directory in the repository containing the source code of the nginx module. (optional)
     
-    --url                       Url used for git clone. (required)
+    --url                       Url used for git clone. You should provide url here or through --env-var-for-url.
+
+    -eu  | --env-var-for-url    Name of environment variable contains url. You should provide url here or through --url.
```

Comment:
Instead of adding a second set of parameters, could you call the script like this?

```sh
./scripts/generate.sh --url $NAP_URL --branch $NAP_BRANCH
```